packages/ajna-contracts/scripts/common/deployment.utils.ts

```diff
@@ -184,16 +184,19 @@ export async function deployPoolFactory(
 
   return { erc20PoolFactory, erc721PoolFactory };
 }
+
+// default rate = 4%
 export async function deployPool(
   erc20PoolFactory: ERC20PoolFactory,
   collateral: string,
   quote: string,
-  deployPools = true
+  deployPools = true,
+  rate = "40000000000000000"
```

Comment:
Rate value has changed from original implementation. Is this correct?

Reply:
Yes, thats our ne default rate : https://discord.com/channels/837076147694207067/1150759720798388254/1180115026107957339